sunpy/sun/constants.py

```diff
@@ -62,222 +62,113 @@
 """
 
 from __future__ import absolute_import, division, print_function
+from astropy.table import Table
+from six import iteritems
```

Comment:
For all sorts of reasons six is bundled to sunpy and thus should be imported from ``sunpy.extern.six``.
e.g. see discussion in https://github.com/sunpy/sunpy/pull/1449